sopel/irc.py

```diff
@@ -189,11 +173,11 @@ def run(self, host, port=6667):
         try:
             self.initiate_connect(host, port)
         except socket.error as e:
-            stderr('Connection error: %s' % e)
+            LOGGER.exception('Connection error: %s', e)
             self.handle_close()
 
     def initiate_connect(self, host, port):
-        stderr('Connecting to %s:%s...' % (host, port))
+        LOGGER.info('Connecting to: %s:%s', host, port)
```

Comment:
I think this was more correct without the added `:`. I also rather like the trailing `...` because registering an IRC connection is a process that may take some time, and this log line is only the start of it.

```suggestion
        LOGGER.info('Connecting to %s:%s...', host, port)
```